eq-author-api/src/businessLogic/onPageDeleted.js

```diff
@@ -8,6 +9,12 @@ const onPageDeleted = (ctx, removedPageId) => {
       return;
     }
 
+    const elseRoute = page.routing.else;
+    if (elseRoute.pageId === removedPageId) {
+      elseRoute.pageId = null;
```

Comment:
Probably worth just ensuring that the sectionId is nulled out here as well.

```js
page.routing.else = {
  sectionId: null,
  pageId: null,
  logical: NEXT_PAGE
}

Reply:
sectionId now also nulled